consensus/types/src/deposit_request.rs

```diff
@@ -36,7 +36,7 @@ impl DepositRequest {
             pubkey: PublicKeyBytes::empty(),
             withdrawal_credentials: Hash256::ZERO,
             amount: 0,
-            signature: Signature::empty(),
+            signature: SignatureBytes::empty(),
```

Comment:
Should we change the type to SignatureBytes for the other request types as well for consistency?

Reply:
curious why we did this?

Reply:
Because we need to allow for invalid bls signature bytes to be sent over the engine api. Otherwise, it becomes a deserialization error when we get the payload from the EL instead of an invalid deposit during processing